pulpcore/cli/rpm/distribution.py

```diff
@@ -60,6 +60,7 @@ def distribution(ctx: click.Context, pulp_ctx: PulpContext, distribution_type: s
 update_options = [
     click.option("--base-path"),
     click.option("--publication"),
+    click.option("--repository", callback=_repository_callback),
```

Comment:
This is def more than just adjusted tests.

Reply:
Well the option was missing, so I added it.

Reply:
And it's needed for the autopublish operations?
Then we can reflect this as a feature in the changelog. Like "Adjusted autopublish to new version of ...".